cmd/template.md

```diff
@@ -29,9 +29,9 @@ function toTop() {
 
 ## Overview
 
-The Open Source Project Security (OSPS) Baseline is a set of security criteria that projects should meet to demonstrate a strong security posture.
-The criteria are organized by maturity level and category.
-In the detailed subsections you will find the criterion, rationale, and details notes.
+The Open Source Project Security (OSPS) Baseline is a set of security controls that projects should meet to be considered secure.
```

Comment:
This line needs to be reverted as noted in #191

Reply:
```suggestion
The Open Source Project Security (OSPS) Baseline is a set of security criteria that projects should meet to demonstrate a strong security posture.
```